docs/docs/tutorials/get-started/customize-sign-in-experience.mdx

```diff
@@ -43,13 +44,79 @@ Toggle on "Enable terms of use" and enter the URL to enforce this process. Logto
 
 ### Language
 
-In most cases, it'll be fine to keep the default value unless:
+Logto provides six predefined languages (v1.0.0-beta.11). You can customize values of invidual keys.
 
-- You want Logto to fall back to a different language when no language matches the user's system preference, or;
-- You want Logto to fix a language to display.
+In addition, it provides 113 language tags. You can create an added language , bring your own translation, and apply to your product UI.
+
+:::note
+
+Currently we only support language customization for Sign in UI. This langauge settings doesn’t apply to the admin console language setting.
+
+:::
+
+#### Terminology
+
+| Terminology                                                                                 | Notes                                                                                                                                    |
+| ------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| Default language                                                                            | When auto-detect is on, the default language will be used when the detected user language isn’t covered in the current language library. |
+| When auto-detect is off, the default language is the only language your software will show. |
+| Logto source values                                                                         | It refers to the values which Logto provided and haven’t been customized by users.                                                       |
+| Custom values                                                                               | It refers to the values which AC users can change for individual keys.                                                                   |
+| Keys                                                                                        | It refers to the identifier of each locale value.                                                                                        |
```

Comment:
instead of having a table, i think it'll be better to use a vivid way to explain the terms and describe the relations between them.

or, can you use a screenshot of the customize language modal and point out what is what?

Reply:
let's keep in mind of JTBD instead of a tech doc. you can talk to guamian about it

Reply:
Yeah, we have a screenshot attached. It's also great to include a visual to explain all the terms.